apps/web-customer/src/components/blocks/LandingPageBlock/HomeBlock/Pricing/PricingComponent/PricingDetail.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { NCLEXBlueLogo } from "core-library/assets";
+import Image from "next/image";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { Box, Toolbar } from "@mui/material";
+
+import { EvaIcon } from "core-library/components/EvaIcon";
```

Comment:
this is enough since evaIcon is imported from the components index.ts
```suggestion
import { EvaIcon } from "core-library/components";
```